README.md

```diff
@@ -174,19 +188,33 @@ kubectl create configmap docker-config --from-file=/$HOME/.docker/config.json -n
 
 ## Usage
 
+### Option 1 - On clusters with existing PVCs
+
 Execute the following to start a pipeline run
 
 ```bash
 # example (using the node-observability-operator)
 tkn pipeline start pipeline-dev-all \
 --param repo-url=https://github.com/openshift/node-observability-operator \
 --param repo-name=node-observability-operator \
---param base-image-registry=quay.io/<your-repo-id>
---param bundle-version=v0.0.1
+--param base-image-registry=quay.io/<your-repo-id> \
+--param bundle-version=v0.0.1 \
 --workspace name=shared-workspace,claimName=pipeline-pvc-dev
+```
+
+### Option 2 - Kind clusters, or without existing PVCs
 
+```bash
+# example (using the node-observability-operator)
+tkn pipeline start pipeline-dev-all \
+--param repo-url=https://github.com/openshift/node-observability-operator \
+--param repo-name=node-observability-operator \
+--param base-image-registry=quay.io/<your-repo-id> \
+--param bundle-version=v0.0.1 \
+--workspace name=shared-workspace,volumeClaimTemplateFile=workspace-template.yaml
```

Comment:
I get an error here. file not found. use the full path i.e `manifests/tekton/pipelineruns/workspace-template.yaml`

Reply:
add` -n operator-pipeline` to the command